Bangazon/Views/Products/Create.cshtml

```diff
@@ -14,7 +14,7 @@
             <div asp-validation-summary="ModelOnly" class="text-danger"></div>
             <div class="form-group">
                 <label asp-for="Product.Description" class="control-label"></label>
-                <input asp-for="Product.Description"Description" class="form-control" />
+                <input asp-for="Product.Description" Description" class="form-control" />
```

Comment:
The `Description"` after `asp-for="Product.Description"` needs to be removed.
```suggestion
                <input asp-for="Product.Description" class="form-control" />
```

Reply:
fixed